src/helm/clients/openrouter_client.py

```diff
@@ -0,0 +1,34 @@
+import os
+from typing import Optional
+from helm.clients.openai_client import OpenAIClient
+
+
+class OpenRouterClient(OpenAIClient):
+    def __init__(
+        self,
+        tokenizer,
+        tokenizer_name,
+        cache_config,
+        api_key: Optional[str] = None,
+        model_name: Optional[str] = None,
+        output_processor: Optional[str] = None,
+    ):
+        self.api_key = api_key or os.getenv("OPENROUTER_API_KEY")
+        self.base_url = "https://openrouter.ai/api/v1/"
+        super().__init__(
+            tokenizer,
+            tokenizer_name,
+            cache_config=cache_config,
+            output_processor=output_processor,
+            base_url=self.base_url,
+            api_key=self.api_key,
+        )
+        self.model_name = model_name
+
+    def _get_model_for_request(self, request):
+        if self.model_name:
+            return self.model_name
+        elif len(request.model_deployment.split("/")) > 1:
+            return request.model_deployment.replace("openrouter/", "")
```

Comment:
If you're following the HELM convention where model name is `mistralai/mistral-medium-3.1` (see the later suggested change), you can just use `request.model_name` here.